@types/modules/stpm/minievents.d.ts

```diff
@@ -0,0 +1,33 @@
+declare interface METMMiniEventUpdateRequest {

```

Comment:
Heads up! It's the "Mini Events and Workshops Module" (ME**W**M). Also, I think that these are just referred to as "Events" and not "Mini Events" as they can also include things like deadlines or meals.
```suggestion
declare interface MEWMEventUpdateRequest {

```